server/src/main/java/org/elasticsearch/search/fetch/subphase/FieldFetcher.java

```diff
@@ -42,12 +42,10 @@
  * A helper class to {@link FetchFieldsPhase} that's initialized with a list of field patterns to fetch.
  * Then given a specific document, it can retrieve the corresponding fields from the document's source.
  */
-public class FieldValueRetriever {
-    public static FieldValueRetriever create(
-        MapperService mapperService,
-        SearchLookup searchLookup,
-        Collection<FieldAndFormat> fieldAndFormats
-    ) {
+public class FieldFetcher {
+    public static FieldFetcher create(MapperService mapperService,
+                                      SearchLookup searchLookup,
+                                      Collection<FieldAndFormat> fieldAndFormats) {
```

Comment:
When we finally format the code with the auto-formatter it'll look like the left instead of the right.

Reply:
This is good to know !